rust/agama-lib/share/profile.schema.json

```diff
@@ -576,7 +605,8 @@
     "sizeString": {
       "title": "Human readable size (e.g., '2 GiB')",
       "type": "string",
-      "pattern": "^[0-9]+(\\.[0-9]+)?(\\s*([KkMmGgTtPpEeZzYy][iI]?)?[Bb])?$"
+      "pattern": "^[0-9]+(\\.[0-9]+)?(\\s*([KkMmGgTtPpEeZzYy][iI]?)?[Bb])?$",
+      "examples": ["2 GiB", "1.5 TB", "1TIB", "1073741824 b", "1073741824"]
```

Comment:
BTW, here the `examples` feature really shines because it provides an alternative to the dense regexp

Unfortunately, `agama profile validate` does not check whether the schema is internally consistent and uses only valid examples. Like, I could use a number here and it would not notice.